cli/go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/authorhealth/go-elation/cli
```

Comment:
I don't think this needs to be a separate module

Reply:
Fair enough. My thinking was by keeping this as a separate module, it avoided making consumers of the client pull in the CLI-specific dependencies like cobra and spew.

Reply:
I think it's OK for now - maybe revisit when we feel the need to?

Reply:
Probably my lack of go-fu, but I'm getting import cycles trying to move this out of the separate package. I'm going to leave this as-is, but if you know a way to resolve this cycles, I'd be happy to refactor.